homeassistant/components/sms/sensor.py

```diff
@@ -16,47 +16,28 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     gateway = hass.data[DOMAIN][SMS_GATEWAY]
     entities = []
     imei = await gateway.get_imei_async()
-    name = f"gsm_signal_imei_{imei}"
-    entities.append(
-        GSMSignalSensor(
-            hass,
-            gateway,
-            name,
-        )
-    )
+    entities.append(GSMSignalSensor(hass, gateway, imei))
```

Comment:
No need for a local entities list here since it's only 1 you can just call:
`async_add_entities([GSMSignalSensor(hass, gateway, imei)], True)`